assets/js/base/components/cart-checkout/form-step/index.js

```diff
@@ -11,12 +11,14 @@ import './style.scss';
 
 const StepHeading = ( { title, stepHeadingContent } ) => (
 	<div className="wc-block-checkout-step__heading">
-		<h4 aria-hidden="true" className="wc-block-checkout-step__title">
+		<h2 aria-hidden="true" className="wc-block-checkout-step__title">
```

Comment:
Should these use the new `Title` component introduced in #2486 ?

Reply:
Related to: https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2486#discussion_r429275237

I think `Title` should be an opinionated block only used when we want it to appear with those specific styles. For headings with other styles, I don't think there is any benefit on using `Title` over native `h1`, `h2`... tags.

Reply:
The comment I [left here](https://github.com/woocommerce/woocommerce-gutenberg-products-block/pull/2486#discussion_r429296378), applies here as well. 

So from your comment, I think you are suggesting here that you don't think the step headings should be linked to the style tree inherited from `Title`?

Reply:
Actually, after seeing the new designs of the _Cart_ and _Checkout_ blocks, it looks like the _Checkout_ step titles and the _Cart_ title will have the same styles, so contradicting my previous comment, I do think this is a good use-case to use `Title` here.

Will update this PR accordingly after #2486 is merged.

Reply:
I have been experimenting with this and I'm not sure the same styles will work, I will discuss this with @LevinMedia, but I would like to decouple it from this PR.

I created a follow-up issue to investigate if the `Title` component can be used in the _Checkout_ block (#2562) so this PR is not blocked by this.